CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
 - FIX Service not found for tokens coming from a trust.
-- FIX XAuth Token not checked if validation is off (#197)
\ No newline at end of file
+- FIX XAuth Token not checked if validation is off (#197)
+- ADD Tests for the IOTAgent plugin (using the generic REST plugin).
```

Comment:
Strictly speaking probably this line shouldn't needed, as tests are part of the "developers tool" and this changelog is mostly intented for final users.

However, it is better to "fail" by exceed than by default, so it's ok with the new line.

NTC